src/s1g_redflag.txt

```diff
@@ -0,0 +1,85 @@
+52644
```

Comment:
List of fids in germany sentinel 1 dataset containing inf/nan.  No access to test set, so can't remove them from test set.